eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -295,6 +297,24 @@ public List<Tojo> select(final Predicate<Tojo> filter) {
         };
     }
 
+    /**
+     * Check the internet connection.
+     *
+     * @return Is the internet connection available
+     */
+    protected static boolean online() {
```

Comment:
@mximp we have `WeAreOnline` class already, check it out

Reply:
@yegor256 made some refactoring around `WeAreOnline`.